monai/networks/nets/magnus.py

```
@@ -0,0 +1,808 @@
+            dim_feedforward=int(hidden_dim * mlp_ratio),
+            dropout=dropout,
+            activation="gelu",
+            batch_first=True,
+            norm_first=True,
+        )
+        self.transformer = nn.TransformerEncoder(encoder_layer, num_layers=depth, enable_nested_tensor=False)
+
+        # Layer normalization
+        self.norm = nn.LayerNorm(hidden_dim)
+
+    def _interpolate_pos_encoding(self, x: torch.Tensor, num_patches: int) -> torch.Tensor:
```

Ruff (0.14.14)

Unused method argument: `x`

(ARG002)